app/views/catalog/_show_availability_sidebar.html.erb

```diff
@@ -19,15 +19,7 @@
   <div class="location--panel location--linked">
     <div class="panel panel-default">
       <div class="panel-heading"><%= t('blacklight.holdings.linked') %></div>
-      <div class="panel-body">
-        <% linked_records.each do |records| %>
-          <ul>
-            <% records.each do |link| %>
-              <li><%= link %></li>
-            <% end %>
-          </ul>
-        <% end %>
-      </div>
+      <%= linked_records %>
```

Comment:
Is a helper appropriate for this? It seems rather than a global function, having something like what was previously here but doing `<% document.linked_records.each` would be clearer. Just a personal opinion though.

Reply:
It would be even clearer if you could just do:

```ruby
render partial: 'show_linked_records', locals: { linked_records: document.linked_records }
```

Reply:
This has been restructured as https://github.com/pulibrary/orangelight/pull/1038/files#diff-cce3cb5f9d0085ad25e65b70ff5dcbc9R22